teuthology/suite/matrix.py

```diff
@@ -196,7 +196,7 @@ def index(self, i):
 
     def tostr(self, depth):
         ret = '\t'*depth + "Concat({item}):\n".format(item=self.item)
-        return ret + ''.join([i[1].tostr(depth+1) for i in self.submats])
+        return ret + ''.join([i.tostr(depth+1) for i in self.submats])
```

Comment:
i think `PickRandom` suffers from the same problem. could you help fix it as well?

Reply:
Done